From 4c11a7daaec0a533f90f8820df8e5d3a0654a0a4 Mon Sep 17 00:00:00 2001
From: Joshua Strobl <joshua@streambits.io>
Date: Sun, 27 Jun 2021 21:51:52 +0300
Subject: [PATCH 1/1] Fix stx_mnt_id usage.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Resolves compilation error ../tools/virtiofsd/passthrough_ll.c:950:36: error: ‘struct statx’ has no member named ‘stx_mnt_id’; did you mean ‘stx_uid’?
---
 tools/virtiofsd/passthrough_ll.c | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/tools/virtiofsd/passthrough_ll.c b/tools/virtiofsd/passthrough_ll.c
index 49c21fd855..f03987953c 100644
--- a/tools/virtiofsd/passthrough_ll.c
+++ b/tools/virtiofsd/passthrough_ll.c
@@ -946,11 +946,7 @@ static int do_statx(struct lo_data *lo, int dirfd, const char *pathname,
             statbuf->st_ctim.tv_sec = statxbuf.stx_ctime.tv_sec;
             statbuf->st_ctim.tv_nsec = statxbuf.stx_ctime.tv_nsec;
 
-            if (statxbuf.stx_mask & STATX_MNT_ID) {
-                *mnt_id = statxbuf.stx_mnt_id;
-            } else {
-                *mnt_id = 0;
-            }
+            *mnt_id = 0;
             return 0;
         } else if (errno != ENOSYS) {
             return -1;
-- 
2.32.0

